windowslib/src/main/resources/scala-native/windows/winnls.c

```diff
@@ -0,0 +1,31 @@
+#if defined(_WIN32) || defined(WIN32)
+#define WIN32_LEAN_AND_MEAN
+#include "Windows.h"
+
+PCWSTR scalanative_win32_LocaleName_Invariant() {
```

Comment:
```suggestion
PCWSTR scalanative_win32_locale_name_invariant() {
```
We standardized these to be all lower case the same as the macro. I changed the whole codebase in a prior PR. Is the `_win32_` required for scoping. If so that would be a new rule.

Reply:
Same for the others.

Reply:
Sorry, I've forgotten about these rules. `_win32_` was added to make sure that we don't get into name clashes with functions defined in other projects, eg. in posixlib, however, it's very unlikely that it would happen, especially since now most of the sources is guarded with preprocessor conditions. 
Most of the existing code in windowslib does contain the `_win32_` part already, if we would like to remove it we should do it before the release in dedicated PR

Reply:
I certainly am not opposed to keeping the `_win32_`.  The upside is that it will never conflict with the other platform. The downside it is another rule but pretty obvious at that.

Reply:
Also, I'm wondering should we create a separate naming schema for windows functions, since Windows API uses much more verbose naming convention with long names using PascalCase, where on Unix most of the functions/macros/constants are rather short.

Reply:
When we create extern functions in clib and posixlib we just used exactly how is it in C because that is the API. If it is toggle case in Windows it should remain that way because that is what the Windows C programmer knows.

The constant helper functions are in normal C style(snake case) `foo_bar` for example just to be consistent - those are only in the annotation so having all those standard seemed to right thing because we started seeing tons of different styles. Does that make sense? We did discuss prefix and length but decided against doing anything because the names in modern C can be very long. cc/ @LeeTibbert 

This was the PR - https://github.com/scala-native/scala-native/pull/2159 - Note we removed the `_libc_ but that is because we know `clib` and `posixlib` can't conflict. They are like subset superset. Windows can't conflict with `clib` but could conflict with `posixlib`.

Reply:
One other comment. When reviewing your Windows C bindings I assumed you did not change the naming otherwise you would be creating a bunch of helper functions. For ones that are need for pass by value/ref then I think changing to snake case to remain consistent makes sense. Like I said in the PR above maybe we can make interopt better so the helper functions are not needed.